### ISSUE TITLE
keydown function conflict

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#409]
Hi all
an addition I made to increase clicker compatibility worked well but may conflict with accessibility features. I've commented on this in the previous commit https://github.com/thexerteproject/xerteonlinetoolkits/commit/3edf6802617037661d54b1187180256fa4ae1ddd but hoping to discuss with someone to see if there is a solution that works in a way that maintains both features.

**Comment by FayCross** [comment]
I think this is affecting other pages too - yesterday I was testing a new page I've just finished and was changing the page on accident when I was trying to change the option I had selected in a MCQ when using keyboard controls. So it probably affects all similar pages - quiz, mcq etc.

**Comment by ronm123** [comment]
Yes sorry Fay I should have been clearer was just using the button nav as an example. Quick fix is to remove or comment out the key 38 and 40 detection but would be good if we could work out a way to keep both features e.g. only use those keys for page navigation when no other buttons have focus.

**Comment by thexerteproject** [comment]
Perhaps config via url params to define navigation keys
